.github/workflows/ci-cosmwasm-contract.yml

```diff
@@ -3,6 +3,7 @@ name: Test CosmWasm Contract
 on:
   pull_request:
     paths:
+      - .github/workflows/ci-cosmwasm-contract.yml
```

Comment:
I thought I already added all of these paths, but I guess not

Reply:
I don't think these need to be here, IIUC Github should automatically run workflows when their definition changes, but I could be wrong...

Reply:
it doesn't :( . I only added these after the workflows failed to trigger on the first push of this PR